Tools/FilterTestTool/run_filter_test.py

```diff
@@ -51,7 +51,7 @@
     root = Tk()
     root.withdraw()
     root.focus_force()
-    log_file = askopenfilename(title="Select log file", filetypes=(("log files", ".bin .log"), ("all files", "*.*")))
+    log_file = askopenfilename(title="Select log file", filetypes=(("log files", ".BIN .bin .log"), ("all files", "*.*")))
```

Comment:
Any reason your didn't fix .log at the same time?

Reply:
@peterbarker Actually .log is default small. .BIN is all capital by default.
Otherwise I'd add .Log .LOg .LOG .loG .Bin .BIn .bIN ... etc